pkg/state/appender.go

```diff
@@ -820,6 +830,15 @@ func (a *txAppender) appendBlock(params *appendBlockParams) error {
 	if err != nil {
 		return err
 	}
+
+	// write updates into the updatesChannel here
+	// TODO possibly run it in a goroutine? make sure goroutines run in order?
+	if a.bUpdatesExtension != nil {
```

Comment:
I think having the dummy implementation of the "blockchain updates extension" looks simpler than `! = nil` checks, on my mind.

Reply:
What do you think?

Reply:
It depends! For tests it was easier to send a nil, but maybe I should do what you suggested. How would you check if it's empty?